TileStache/__init__.py

```diff
@@ -57,6 +57,20 @@
 
 from ModestMaps.Core import Coordinate
 
+# python3 compatibity while retaining checking
+# for both str and unicode in python2
+try:
```

Comment:
It is also confusing to have this here. instead we can have a separate module called py3_compat.py
We would later have this changed to favor python 3 style.

Reply:
This makes perfect sense, I'll get it done. If you agree, I'll just dump all the compatibility related code there and import as needed.

Reply:
perfect